charts/telegraf-operator/values.yaml

```diff
@@ -1,6 +1,7 @@
 replicaCount: 3
 image:
   repository: quay.io/influxdb/telegraf-operator
+  tag: ""
   pullPolicy: IfNotPresent
   sidecarImage: "docker.io/library/telegraf:1.14.4"
```

Comment:
This sidecar Image is quite old.  Latest Telegraf is now 1.19.2

Shall we update this along with the PR?